rever/activities/conda_forge.xsh

```diff
@@ -104,6 +104,8 @@ class CondaForge(Activity):
         a pull request to the upstream conda-forge feestock, default True.
     :$CONDA_FORGE_RERENDER: bool, whether the activity should rerender the
         feedstock using conda-smithy, default True.
+    :$CONDA_FORGE_FORK: bool, whether the activity should create a new fork of
+        the feedstock, default True.
```

Comment:
This needs to be more clear that a fork will be created only if it doesn't exist already